content/rs/developing/crdbs/developing-hll-crdb.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: Developing with HyperLogLog in a CRDB
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+**HyperLogLog** is an algorithm to address the [count-distinct problem](https://en.wikipedia.org/wiki/Count-distinct_problem).
+To do this it approximates the numbers of items in a [multiset](https://en.wikipedia.org/wiki/Multiset).
+Determining the _exact_ cardinality of a multiset requires memory according to the cardinality of the multiset.
+Because it estimates the cardinality by probability, the HyperLogLog algorithm can run with more reasonable memory requirements.
```

Comment:
Don't we have a rule of thumb number for how much less?
Personally I would say HyperLogLog again in the beginning of the sentence to emphasize it. (" Because HyperLogLog estimates the cardinality...") but your call, you're the tech writer.

Reply:
Please replace all occurrences of the term "multiset" with "set". AFAICT, "multiset" is an overly formal term that's never used in any of the canonical Redis docs. Using "multiset" results in confusion, because no one know what it even means in this context.

Reply:
@banker So you think we should remove the link to wikipedia also?

Reply:
I'd leave the Wikipedia link.

Now that I've read it, I'd drop line 9 altogether. (To do this it approximates the numbers of items in a [set](https://en.wikipedia.org/wiki/Multiset).)